src/GBI.cpp

```diff
@@ -142,6 +143,11 @@ bool GBIInfo::isHWLSupported() const
 	return true;
 }
 
+void GBIInfo::_flushCommands()
```

Comment:
Sure, but MSVC doesn't support extended constexpr, which is why I didn't do that.